dev/devicelab/lib/framework/devices.dart

```diff
@@ -193,6 +194,18 @@ abstract class Device {
   /// Stop a process.
   Future<void> stop(String packageName);
 
+  /// Wait for the device to become ready.
+  Future<void> awaitDevice();
+
+  Future<void> uninstallApp() async {
+    await flutter('install', options: <String>[
+      '--uninstall-only',
+      '-d',
+      deviceId]);
+
```

Comment:
Maybe we need to add a small delay here. If the check is fast enough it will tell the device is available before it becomes unavailable.

Reply:
That makes sense. On the order of seconds?

Reply:
Yeah, let's start with 2s and update as we get more data.

Reply:
Done!